extension/src/experiments/model/util.ts

```diff
@@ -0,0 +1,20 @@
+import get from 'lodash.get'
+import { formatDate } from '../../util/date'
+import { splitColumnPath } from '../columns/paths'
+import { Experiment } from '../webview/contract'
+
+export const getDataFromColumnPath = (
```

Comment:
Since the logic for getting the data in the experiment selection quick pick and the tooltips is pretty similar, I created a helper function to be reused in both places.